community/kernel/src/main/java/org/neo4j/graphdb/factory/GraphDatabaseSettings.java

```diff
@@ -143,6 +144,10 @@
     @Description( "Threshold for rotation of the internal log." )
     public static final Setting<Long> store_internal_log_rotation_threshold = setting("store.internal_log.rotation_threshold", BYTES, "20m", min(0L), max( Long.MAX_VALUE ) );
 
+    @Description("Log level threshold.")
+    public static final Setting<FormattedLog.Level> store_internal_log_level = setting( "store.internal_log.level",
+            options( FormattedLog.Level.class ), "INFO" );
+
```

Comment:
Alright, "internal" in this case refers to the log file, messages.log, being internal. And not the setting. Got it.